Jenkinsfile

```diff
@@ -0,0 +1,78 @@
+pipeline
+{
+    agent
+    {
+        node
+        {
+            label 'embedded-docker'
+        }
+    }
+    environment
+    {
+        MAKEFLAGS = '-j 3'
+        PACKAGE_NAME = 'um_kernel'
+        DOCKER_CONTAINER_NAME = 'um-kernel_build'
+        DOCKER_IMAGE_NAME = 'um-kernel_image'
+        BUILD_OUTPUT_DIR = '_build_armhf'
+    }
+    stages
+    {
+        stage('Prepare')
+        {
+            steps
+            {
+                echo "Starting with a clean environment"
+                step([$class: 'WsCleanup'])
+                checkout scm
+                sh 'git submodule update --init --recursive'
```

Comment:
i've been thinking about this step for a while now, and I think it should be part of our build script/make file so that you can do build src_get. however we cannot do this as we do not want to run the source init from the container itself. not sure how to best solve this yet